WebRoot/templates/prefs/Pages.template

```diff
@@ -425,8 +425,8 @@ and the container element is replaced with that control.
 				<$ var isHtmlCompose = data.isEnabled(ZmSetting.HTML_COMPOSE_ENABLED); $>
 				<$ if (isHtmlCompose) { $>
 					<tr>
-						<td class='ZOptionsLabelTop' valign=top><$=ZmMsg.composeLabel$></td>
-						<td class='ZOptionsNestedTable'>
+						<td class='ZOptionsLabelTop ZmComposeLabel' valign=top><$=ZmMsg.composeLabel$></td>
```

Comment:
can we also name it to indicate its a preference compose section ?  I see below "ZmPrefMailCompose" maybe something similar to that like ZmPrefComposeLabel?

suggestion,
Also below class name (ZmPrefMailCompose) doesn't help understand what it contains . For ex: if its a container, just content we may use that in class names ? ZmPrefComposeContent/ ZmPrefComposeContainer ?